src/tools/js-wrapper.h

```diff
@@ -103,16 +103,20 @@ static std::string generateJSWrapper(Module& wasm) {
     }
     ret += std::string("instance.exports.") + exp->name.str + "(";
     bool first = true;
-    for (const auto& param : func->sig.params) {
+    for (auto param : func->sig.params) {
       // zeros in arguments TODO more?
       if (first) {
         first = false;
       } else {
         ret += ", ";
       }
-      ret += "0";
-      if (param == Type::i64) {
-        ret += ", 0";
+      if (param.isRef()) {
```

Comment:
If we support GC, can this be data or something else too?

Reply:
`isRef()` returns true for a reference to GC data, so this should cover that. (Or did I misunderstand your question?)

Reply:
Ah you're right. Then is this possible to be an RTT?

Reply:
Oh right, it could be... in that case we can't do anything (no way to create an RTT in JS), and the JS side will trap. When I get to adding RTTs as parameters in the fuzzer I'll have to handle that somehow...